### DISCUSSION TITLE
Proof of Humanity

### DISCUSSION THREAD

**Discussion by stevew00dy** [sunflower-land/sunflower-land#1621] [Ideas]
As we all know, BOTS have been a problem in every project where there is a financial incentive to make even the smallest ROI and so this discussion is around a concept to protect the community and the economy at the same time.

Any tradition finance system and also and CEX requires KYC - know your customer. Its purpose is to help prevent money laundering and illegal activity. Maybe in Web3 fear this because they don't trust the entities that hold the data. 

I would like to consider an incentive program that is optional but rewards those we can prove are human.

This consists of two parts

1. Reduction of withdraw tax
2. Increased withdrawal limit

Right now the standard tax is 30%

I suggest a 2 step KYC program that starts with Jigga and Discord or Twitter verification and this would first enable the bank so withdraws can be made

Additional and optionally 

Upload approved ID via a reputable 3rd party to prove who you are and receive a 10% reduction in withdraw tax

In addition to the tax break, I suggest we impose weekly limits on how much a farm can withdraw until they complete the first stage and if they complete the second stage it's unlimited 

A third potential solution is to include a time delay on withdraw of 7 days for stage 1 that is reduced to instant after stage 2.

Anyone can play the game regardless of KYC but the bank (responsible for withdrawal) would be affected by this so only those looking to extract value would need to verify they are not a bot.

**Comment by Dukler** [comment]
I don't like KYC at all, I maybe even stop playing the game if this is part of it, (KYC meaning I have to upload some ID), but if we find a way to prove we are humans without revealing identities I'm up for it. But I don't like the idea of this space going for this direction of I have to upload my ID everywhere and anyone knows who I am. I would approve of the idea of connecting to twitter or maybe my phone number, but that's it. No IDs tho. Just my 2 cents.
edit: typo

**Reply by stevew00dy** [reply]
So firstly I would love to know why you don't like KYC - I'm just curious on the reasons. 

You would never have to KYC to play the game, please understand that playing the game and extracting value from the game are two separate components and for this - I'm talking only about extraction for simplicity. So you would not need to KYC to play. 

Also, when you say everyone would know who you are, it's not public, even the team wouldn't know. It's only the tools used to verify that would store that data. That is even if storage of it is needed, maybe once verified it's deleted. I don't know. But my point is that it's not public unless the KYC provider gets hacked and data gets leaked. But that is the same with anything you use online. 

Steam uses Mobile verification, Some others use Google / Twitter / Discord. There are many ways. I'm not saying any are great or bad. Just exploring the reasons why.

**Comment by wersout** [comment]
I have no problem with do a KYC and in fact i like the ide if it gives a tax break and keeps bots out. The third potential solution is seen in many other sites and well implemented  in our minds as it is so in all I`m fine with all of it..

**Reply by stevew00dy** [reply]
Appreciate your feedback. Overall from polls I've found that the majority don't mind but there are some who do. I would love to hear concerns and then look at all possibilities

**Comment by malkaron** [comment]
I think the idea is great. Everything that contributes to the game being serious and profitable in some way I'm in favor. I believe that the sunflower will be the most reliable game!

**Comment by ShiningMinions** [comment]
I hate KYC, but I really appreciate the amount of thought you have put into this. If KYC is unstoppable, then this is definitely a good starting approach.

Here are my biggest issues for SFL KYC:

1. As a player, I don't understand the impact of the botting problem. This is because I don't have statistics showing me the current impact bots are having on the game. And because I don't see the problem, I don't see why KYC is needed. If you could somehow provide the community clear statistics for the the current botting problem PLUS the impact it's having on the game, then I will 100% help you brainstorm solutions (and collectively, we can see if KYC makes sense).

2. My biggest fear is that KYC in SFL will be a slippery slope. Today what you proposed seems somewhat reasonable, but as time goes on KYC is being mentioned more and more, and restrictions are going up and up. We already have jigger verification but now we need more? My fear is that it will never stop, and there definitely will be a point where I stop playing if it gets too much. If we want to stay true to the decentralized roots of the game, then we need to think outside of the box.

**Reply by stevew00dy** [reply]
Point 1 would be down to Adam. There is a benefit and also a drawback to providing data around bots. We are not hiding anything from the community but we do want to ensure that we don't give bots and abusers data they can use against us. 

With that said - for this request. I'll check with the team and let you know. 

Point 2 - 100% agree and in an ideal world, we wouldn't need anything but there is a duty of care and responsibility to protect the economy and community and as we expand it gets harder. We don't want to do more than we need but we must be upfront and honest that it's a war. Not a Goblin one, but one where people will always try to destroy what you build and we have to protect honest people from the rest.

**Comment by NotKeen** [comment]
Also not a big fan of KYC, I feel like it goes against the core values of crypto (permissionlessness).

I liked the idea I saw somewhere to give more privileges the more value you hold on your farm (NFT value + value of crops / resources). The reasoning would be that a multi or botter would take a great risk leaving high value on a farm that might get banned, so they would self-select to not hold much value on a farm.

Therefore we could give more freedom and less tax to those that choose to hold more value on farm.

Downside to this is that it might be catering to whales too much and might frustrate free players / beginners.

**Reply by eodestroyer** [reply]
I am in favor of restrictions on the least valued and less restrictions on the most valued nft holder.  Get rid of bot checking on those who have a clear track record of playing to own and not playing to earn, IMO.

**Comment by Polysemouse** [comment]
Not a fan of KYC.  If it really has to be implemented for people to extract value, it would be nice if it can be done without the need of an ID.

People can still bot after they pass KYC.

My fear is that the KYC data would fall into bad hands (by data breaches etc.). Imagine one day some bad actors found out who the person is / where the person is living for a Sunflower Land farm that is worth a few hundred thousand dollars.  This already had happened with people with bags of Bitcoin.

A better solution is to implement core game mechanics that would discourage people from botting the game in a financial standpoint compared to people who has well established farms.

**Comment by Life-Is-Great** [comment]
Personally.
I will never give up my identity to anyone in crypto except those that I chose myself and those that I know identity of myself.
_( Only in this case. I would be willing to reveal my true identity to core team as long as it is not being stored in anyway anywhere )_

So to be honest.
By my opinion. This KYC thing is a horrible idea and can be bypassed easier than "**Coded Cheat Preventions**". People go over border with fake identity, people do crimes based on fake identity etc.

If it would come to that I'm forced to do KYC where my personal info/image is sent elsewhere than my own hard drive. I would 101% stop playing and not have same feelings for Sunflower Lands decentralized game as I do today. And I would use legal rights to get what is mine back +extra for the pain of having to go thru all that.
**( Do not misunderstand me plz. I love this game and community and I have proven that but I'm totally against this KYC thing )**

<br><br>

If you are going to ask todays web3 users for **KYC** on a **decentralized** platform then I do think that it would be best to build an **alternative** for people that want to play **centralized**.

There are other ways. Just look at big companies and how they do it and learn.
If still chosen to force KYC I do believe that best way would be to inform all players at least 30 days before this happens so that everyone can react to this in a way that fits them.


<3 All love to SFL.
All hate to KYC...

**Reply by Myth727** [reply]
Completely understandable all these factors,  it's funny for me I'm fine with KYC but my wife and her farm. She would never, leaving many assets and funds to waste. There does need to be a mid point/workaround.

**Reply by Life-Is-Great** [reply]
I understand that this is coming from good intentions.
_(To try prevent bots from stealing normal players wealth and sabotage game)_

I also know for a fact that stopping bots from playing is bringing less $ to the team (Because team gets payed from syncs)
With this in mind.
It proves that the team is not selfish and that the team is for real trying to stop those gameplay sabotaging exploits.

But I do believe that KYC would bring more bad than good to the game. Imagine loosing "Decentralized" aspect from Sunflower Land title...

**Comment by IvanSkazkin** [comment]
I will leave the project. I invested $1200+. But I will leave if there is KYC. The point of decentralization is its anonymity. And your task is to think how to solve it. You can make it non-mandatory. Let's get more bonuses for those who want to do on one's own KYC (remove withdrawal tax). Or if a person was banned for suspicion and he has a lot of valuable NFT. Let him prove it. But doing it for everyone is not smart.

I am against KYC.

I've heard so many words from you that expanding the land will solve the problem of bots, and then I find out that you do not have a clear plan.

Also twitter is banned in my country (not my fault). Doesn't let me into the discord. I got kicked off the server and can't log in anymore. I wrote to those support, they said that I would change my nickname and they would help me. I waited 10 days. After that, the appeal was simply closed. I still can't log into discord. And how can I even look at the future of the game after your statements?


A little about what would attract me, but maybe this is a problem for you.
I like that the frogs are not on the farm, but fly through the air. I'm not afraid of a ban, because I can sell it. Even if I get banned. After all, there is the Presumption of Innocence, which means that it is not possible to ban only bots, sometimes you ban a person who has a lot of expensive nft for $2000+ dollars and he was not guilty. This is the main problem for me in SFL. If you did KYC and after that people had the opportunity to store NFT over the air, I think it would attract players more

Give those who will make KYC the opportunity not to keep NFT on the farm but over the air. For example, I have a t3 beaver (I bought it myself). And a week ago they gave me a jigger. I took the beaver out of the farm after verification. And now I'm scared to start. I had a desire to sell all NFT. After all, at any moment I can be banned and I will lose my finances.

Hear me. It is necessary to give privileges such that people themselves would like to participate in this, and not to force everyone.

There is only one step from love to hate, remember this when you enter something important.

I am a big SFL fan. After all, I would not invest in something that I do not like. But your statements scare me.

With love

**Reply by stevew00dy** [reply]
Firstly, nothing is designed here to scare you. This is an open conversation. Nothing more and nothing less. It's a chance to hear perspectives and ideas. 

Secondly, it's not that we don't have a plan or a solution but to just assume everything will be ok and never go wrong or that we will never be attacked / exploited or have people abusing us is simply naïve. We MUST do everything we can whilst things are calm to protect ourselves against when things are crazy. 

There are always unknowns. We can't possibly plan for everything, this is why we adapt and grow. It's a natural part of evolution in life as well as in a project. 

You make some very good points about software limitations by location, these are things that these conversations are designed to open up. With regards to your discord ban - let me check, no ticket would be closed without a resolution so please send me a DM with your ticket number and I'll chase it.

**Reply by IvanSkazkin** [reply]
**https://sunflowerland.freshdesk.com/support/tickets/3578**

Thanks for the feedback. But I'm still afraid to use expensive NFTs after verification. I hope you like my idea with nft in the air like frogs after KYC.

**Comment by stevew00dy** [comment]
So here is another concept to consider around verification. 

When someone starts playing, there is a quest to identify the mysterious bumpkin (thats you) 

Through some lore and game mechanics we can get you to complete several activities

- Link your twitter account
- Setup 2FA
- Link your Google Account
- Provide a Selfie
- Etc etc

These steps would give you boosts and bonuses along the way that will make your starting experience in the game a little easier than someone who was not willing. 

It would be an optional quest, but one that would reward the player for completing a layered approach (I know it's not perfect and above is a guide not final recommendations)

**Reply by wersout** [reply]
Every government in the eu, usa and probably asia has requested investigations on how to regulate crypto and the decentralized one is part of it.. 
As long as those companies pay salaries, taxes or fees they will eventually be regulated.. Now it's about just that they haven't caught up with the legislation but they will.. 
There is no chance that the companies that work with and in the decentralization will not be heavily regulated when it comes to the financial.. It will still be decentralized data that nobody really own but as long as you make a profit on it it will be regulated.. 
Then if KYC is the best solution it remains to be seen but it will not look like it does now.. It is extremely important to talk about because it is inevitable that we are going that way and it is perhaps best if the industry finds a way that governments can be satisfied with rather than them setting the rules. 
As long as they see that there is an opportunity for criminal activity, it will be regulated in one way or another.. I have read about different solutions in the last year, everything from a personal token to KYC, but it would be best if the industry came up with a smart solution that can work well in the decentralized web3.. Just my thoughts and about what I read recently....

**Comment by Theadra** [comment]
Thinking along the lines of blockchain tech, I have this really loose idea of securing this game's player base. 

Enlist players who want to be entrusted with verifying players via whatever social media platform is useful/safe that year, these "validators" will provide a stake for their role, they will receive some of the on-boarding costs when a new player joins. If a validator falsifies a new player, their stake gets slashed a certain percentage and/or revoked. If a possibly cheating player gets reported, those validators get paid a small fee to re-validate accounts. Reports can only come accounts not associated with any one validator, a single validator in theory should never see the same account within a preset period again. Other validators will also have a second follow-up verification for a player, this is how we ensure other validators are following protocol. 

This is similar to like a discord mod, but we let the community assist with policing the player base, while giving players a chance to earn some SFL via the treasury. But also offsetting bad actors with an insurance pool. This will be slow at first, but as the game gains more attention this will be vital in policing the large community.  I didn’t include more details to keep this short as possible.

There are still issues with this, like;

Holy geez this seams like a lot of programming or maintenance, possible conflicts of interest, this may be worse than just KYC (Traditional), and this may be just overkill. My thought is, we entrust the community with it’s own security, held up by smart contracts/whatever/stake’s, this gives the community another way to support this platform, but also reducing load on the core team. 

Please excuse my grammar, I wrote this lit AF at 1am.

**Reply by Theadra** [reply]
My thoughts are, when any form of KYC involves storing data-thats-on-a-goverment-ID, I don’t see it as decentralized,
I see two paths SFL will eventually take, 

Easy, dropping the “decentralized” title and moving on, its just a game.

“OR” 

Hard, sticking to the meaning and forging new tools for privacy preserving player verifications.

Some countries have made crypto illegal, theses players will look for decentralized platforms, but so are bot armies. Maybe a middle ground is achievable. I personally look for projects that support this decentralization, I also understand we’re still figuring this out. At the end of the day, you’ll always have an audience of some kind, you don’t need players like me, you need the players who enjoy a calm, relaxing farming sim. Your idea of KYC is a great start, but I would drop word decentralized moving forward.

Also, for PR sake, stop using KYC to describe verifying players, people instantly think what exchanges do with KYC data, give it to the government. My 2 SFL’s

**Comment by goingsnginx** [comment]
Proof of Humanity is indeed very important. In fact, you don't need to prove your identity, but you only need to prove that you are a personal person

However, what must be acknowledged is that because of the existence of interests, the robot account can never be killed. All we can do is to continuously increase its cost and threshold.

We need some strategies to allow bot competition to increase their costs at the same time

1.Score for user behavior
There are different behavior points for game activities, such as tree cutting, mining, pasture, etc.
for example:
Farming: 1
Egg: 1
Tree chop: 2
Mining: 3
User quality = points / behavior frequency
Can add some grace period to newly created land

2.Some additional verifications of 5% of users with the lowest score, other top 95%of the users are not affected

3.Through the verification of the second step, we can find some suspicious behaviors, popping up a face recognition or directly banned the suspicious account

4.The banned account needs to go to Discord to appeal, because the number of bots is large, and the appeal will be very difficult.

Most bots have a simple behavior and do not participate in the game in depth, so they usually have a low score.

If the bot doesn't want to be the last 5% and doesn't want to be verified or banned, they need to improve the depth of the game and improve their technology, which forces the robot account to compete with each other

**Reply by Myth727** [reply]
Like the ideas, but bluntly not all users can use discord. I used to be a Moderator for 6 groups, but my wife decided discord is not healthy for a marriage.  So this would have to be adjusted alot also.

**Comment by celpower** [comment]
I don't like the idea of KYC at all, not because I have anything to hide but because I don't get to chose who will I be giving my info to. I have had opportunities to buy crypto thru different sites that required KYC and since I was not familiar with the company i decided not to do business with them. I did work with others that required KYC but it was my choice.
A bit ago Jigga was implemented, which is a company most never heard of and didn't even have a decent website so we cold try to learn who they were and their business. And if you think now KYC is needed that means that Jigga is not solving the problem. We had goblins stealing shovels, chests, click on 3 goblins, still not working? Anyone can pass KYC and go back to using bots.
So we will be forced to give God knows who our sensitive info to probably another company no one ever heard of so it "looks" like we will take care of bots and have our sensitive info available for hackers to feed off of.
I guess this would have not been an issue had I know this was a requirement to play a game before I started  and I could have made a decision before spending money and time.
I hope should KYC be implemented that a known company is picked and not another company with an unfinished website and that this won't scare current and future players.

**Reply by ShiningMinions** [reply]
> And if you think now KYC is needed that means that Jigga is not solving the problem. We had goblins stealing shovels, chests, click on 3 goblins, still not working? Anyone can pass KYC and go back to using bots.

I couldn't agree with you more. The issue is that we have no metric to measure success/failure of this problem. It's like playing football without ever keeping score. How do we know if Jigger is working? How do we know that Goblin stealing shovel is working? As a community, let's agree on some metric which clearly **measures this problem first**. Then, when we implement a solution (i.e. KYC), we can then go back to this metric, and clearly see if the solution is working or not.
